### ISSUE TITLE
A11y_Model Builder_Add Data_Input_Non-Text Contrast: Focus has ratio 1.37 over the combo box menu items of “column to predict” and “Time series column”.

### ISSUE THREAD

**Issue by manaswita-chougule** [dotnet/machinelearning-modelbuilder#2183]
### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Visual Studio Client; #SH-ModelBuilder-Win32-Jun2022; #Win32; #WCAG1.4.11; #Non-textContrast;

### Environment Details:
Application Name: Model Builder
Windows Version: Windows 11 22H2 OS Builder 25145.1011
Visual Studio: Microsoft Visual Studio Enterprise 2022 (64-bit) - Preview Version 17.3.0 Preview 2.0

### Repro Steps:

1. Install Visual studio and open Visual studio
2. Navigate to C# project name and right click on it
3. Navigate to "Add" button and press enter
4. Navigate to "Machine learning model" and press enter
5. "Add new item" popup will open
6. Navigate to "Add" button and press enter
7. "Select scenario" popup will open and select "Forecasting" scenario
8. "Select training environment" page will open then select "Next step"
9. "Add Data" page will appear tab till Input section.
10. Tab till “column to predict” and “Time series column” and press enter.
11. Check whether the Focus has ratio 1.37 over the combo box menu items of “column to predict” and “Time series column” or not.

### Actual Result:
Focus has ratio 1.37 over the combo box menu items of “column to predict” and “Time series column”.

### Expected Result:
Focus should have ratio greater than or equal to 3:1 over the combo box menu items of “column to predict” and “Time series column”.

![A11y_Model Builder_Add Data_Input_Non-Text Contrast Focus has ratio 1 37 ](https://user-images.githubusercontent.com/91586101/176695207-97f55159-d49a-4cd5-8b24-b19ec8c47770.png)

**Comment by beccamc** [comment]
@manaswita-chougule I'm confused about this issue. It should be the comparison between the black text of "Low" and the blue highlight right? Where is the white #FFFFFF color coming from?  When I look at this contrast I see 12.1 

![image](https://user-images.githubusercontent.com/9122518/180484024-24b088ba-dcc8-4108-b416-64dc5bb4414f.png)

**Comment by manaswita-chougule** [comment]
Hi @beccamc, the contrast ratio of the focus is passing the minimum required criteria of non-text contrast. Hence closing the bug. Please find attachment.
Thanks!

![2183_Fixed](https://user-images.githubusercontent.com/91586101/180744409-f7cf842d-69a2-4497-b5f0-b277ce0ceadf.png)
